rest/plugins/auth0_logincounts.py

```diff
@@ -0,0 +1,134 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+# Copyright (c) 2014 Mozilla Corporation
+
+import requests
+import json
+import os
+import sys
+from configlib import getConfig, OptionParser
+from datetime import datetime, timedelta
+from mozdef_util.elasticsearch_client import ElasticsearchClient, ElasticsearchInvalidIndex
+from mozdef_util.query_models import SearchQuery, TermMatch, TermsMatch, QueryStringMatch, RangeMatch, Aggregation, ExistsMatch
+from mozdef_util.utilities.toUTC import toUTC
+from mozdef_util.utilities.logger import logger, initLogger
+
+
+class message(object):
+    def __init__(self):
+        '''register our criteria for being passed a message
+           as a list of lower case strings to match with an rest endpoint
+           (i.e. blockip matches /blockip)
+           set the priority if you have a preference for order of plugins
+           0 goes first, 100 is assumed/default if not sent
+
+           Plugins will register in Meteor with attributes:
+           name: (as below)
+           description: (as below)
+           priority: (as below)
+           file: "plugins.filename" where filename.py is the plugin code.
+
+           Plugin gets sent main rest options as:
+           self.restoptions
+           self.restoptions['configfile'] will be the .conf file
+           used by the restapi's index.py file.
+
+        '''
+
+        self.registration = ['logincounts']
+        self.priority = 5
+        self.name = "auth0LoginCounts"
+        self.description = "count failures/success logins"
+
+        # set my own conf file
+        # relative path to the rest index.py file
+        self.configfile = './plugins/auth0_logincounts.conf'
```

Comment:
You reference a config file here, but there isn't one included as part of the PR?

Reply:
If we're not using the config file anywhere, nor are we using the 'ignoredusers' key from a config file, can we just remove the logic to load a config file entirely?

Reply:
we can, but I'll bet we add it right back in the next revision depending on what sort of data shows up. In some cases it seems like auth0 is sending " " as a username for example.

Reply:
I think that's fine, we can cross that bridge when we get there.